modules/io/include/visp3/io/vpVideoReader.h

```diff
@@ -325,6 +327,23 @@ class VISP_EXPORT vpVideoReader : public vpFrameGrabber
       this->lastFrame = last_frame;
     }
 
+    /*!
+      Sets the frame step index.
+	  Throws exception if the input step is less than 1.
+	  The default frame step is 1
+
+	  \param frame_step : The frame index step.
+
+	  \sa setFrameStep()
+	*/
+	inline void setFrameStep(const long frame_step) { // important change
+		if (frame_step < 1) {
+			throw(vpException(vpException::badValue,
+			    "The frame step should be more than 0"));
+		}
+		this->frameStep = frame_step;
+	}
+
```

Comment:
Moreover `setFrameStep()` should be modified to allow negative steps:
- in the doc remove `Throws exception if the input step is less than 1.`
- in the code remove the exception
````
		if (frame_step < 1) {
			throw(vpException(vpException::badValue,
			    "The frame step should be more than 0"));
		}
````